ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -558,10 +558,9 @@ def _wan_control_plane_profile_name(self) -> str:
     def _wan_control_plane_application_profile_name(self) -> str:
         """
         Control plane application profile name
-
-        TODO: make this configurable
         """
-        return "APP-PROFILE-CONTROL-PLANE"
+        control_plane_virtual_topology = get(self._hostvars, "wan_virtual_topologies.control_plane_virtual_topology", default={})
+        return get(control_plane_virtual_topology, "application_profile", default="APP-PROFILE-CONTROL-PLANE")
```

Comment:
```suggestion
        return get(self._hostvars, "wan_virtual_topologies.control_plane_virtual_topology.application_profile", default="APP-PROFILE-CONTROL-PLANE")
```

Reply:
Hey sorry, somehow missed it in last upload.